EntityFramework.Utilities/EntityFramework.Utilities/MappingHelper.cs

```diff
@@ -170,16 +170,16 @@ public EfMapping(DbContext db)
         private static List<PropertyMapping> GetProperties(IEntityType t, Type clrType, string[] path)
         {
             var basePath = string.Join(".", path);
-            var props = t.GetProperties().Where(p => !p.IsShadowProperty)
+            var props = t.GetProperties().Where(p => !p.IsShadowProperty())
                     .Select(p => new PropertyMapping
                     {
-                        ColumnName = p.SqlServer().ColumnName,
+                        ColumnName = p.GetColumnName(),
                         PropertyName = string.IsNullOrWhiteSpace(basePath) ? p.Name : basePath + "." + p.Name,
-                        DataType = p.SqlServer().ColumnType,
+                        DataType = p.GetColumnType(),
                         ForEntityType = clrType, // Use to get the right type for the whole tree 
                         IsPrimaryKey = p.IsPrimaryKey(),
-                        IsStoreGenerated = p.SqlServer().ValueGenerationStrategy != null, // Can be null | HiLo | Identity. Only null is not generated
-                        DataTypeFull = p.SqlServer().ColumnType,
+                        IsStoreGenerated = p.GetValueGenerationStrategy() != SqlServerValueGenerationStrategy.None,
```

Comment:
I don't really know what this means, but comparing with `SqlServerValueGenerationStrategy.None` made it work.

Reply:
Should be the same. It's if the value is generated by SQL or not. Typical case is an int PK column with an identity seed.